test/Microsoft.IdentityModel.Logging.Tests/LoggerTests.cs

```diff
@@ -237,7 +238,39 @@ public void PrepareMessageWithNoArguments()
 
              var exception = LogHelper.LogExceptionMessage(new ArgumentException("This is the first parameter '{0}'. This is the second parameter '{1}'."));
         }
-    }
 
+        [Fact]
+        public void EventLevelToEventLogLevelMapping()
+        {
+            var logger = new TestLogger();
+            LogHelper.Logger = logger;
+
+            var errorMessage = "Test exception message";
+            LogHelper.LogExceptionMessage(EventLevel.Error, new ArgumentException(errorMessage));
+
+            Assert.True(logger.ContainsLogOfSpecificLevel(errorMessage, EventLogLevel.Error));
+            Assert.True(logger.ContainsLogOfSpecificLevel("Microsoft.IdentityModel Version:", EventLogLevel.LogAlways));
+
+            var guid = Guid.NewGuid().ToString();
+            LogHelper.LogArgumentNullException(guid);
+
+            Assert.True(logger.ContainsLogOfSpecificLevel("IDX10000:", EventLogLevel.Error));
 
+            var infoMessage = "Test information Message. {0}";
+            var arg = "Test argument.";
+            LogHelper.LogInformation(infoMessage, LogHelper.MarkAsNonPII(arg));
+
+            Assert.True(logger.ContainsLogOfSpecificLevel(string.Format(infoMessage, arg), EventLogLevel.Informational));
+
+            var verboseMessage = "Test verbose Message. {0}";
+            LogHelper.LogVerbose(verboseMessage, LogHelper.MarkAsNonPII(arg));
+
+            Assert.True(logger.ContainsLogOfSpecificLevel(string.Format(verboseMessage, arg), EventLogLevel.Verbose));
+
+            var warnMessage = "Warn Message. {0}";
+            LogHelper.LogWarning(warnMessage, LogHelper.MarkAsNonPII(arg));
+
+            Assert.True(logger.ContainsLogOfSpecificLevel(string.Format(warnMessage, arg), EventLogLevel.Warning));
```

Comment:
could you use [Theory] and [InlineData()] to capture some of the variables so you can use one Assert.True() for all the log messages & have less code duplication.